### PR TITLE
Update lofreq callparallel

### LINKED ISSUE #6426: update module: LOFREQ/LOFREQ_CALLPARALLEL
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by nevinwu**
## PR checklist

Closes #6426 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

Updated `nf-test`.

### CODE REVIEW

File: modules/nf-core/lofreq/callparallel/tests/main.nf.test [hunk 1]
```diff
@@ -34,7 +34,7 @@ nextflow_process {
                 { assert process.success },
                 { assert snapshot(
                     process.out.versions,
-                    process.out.tbi.collect { file(it[1]).getName() }
+                    process.out.tbi
                     ).match() },
                 { assert path(process.out.vcf.get(0).get(1)).linesGzip.contains('##INFO=<ID=CONSVAR,Number=0,Type=Flag,Description="Indicates that the variant is a consensus variant (as opposed to a low frequency variant).">') }
```
Comment (famosab) [inline modules/nf-core/lofreq/callparallel/tests/main.nf.test:39]: When we update the tests anyway, lets also use the vcf nf-test plugin!

```suggestion
```

File: modules/nf-core/lofreq/callparallel/tests/main.nf.test [hunk 2]
```diff
@@ -34,7 +34,7 @@ nextflow_process {
                 { assert process.success },
                 { assert snapshot(
                     process.out.versions,
-                    process.out.tbi.collect { file(it[1]).getName() }
+                    process.out.tbi
```
Comment (famosab) [inline modules/nf-core/lofreq/callparallel/tests/main.nf.test:37]:
```suggestion
                    process.out.tbi,
                    path(process.out.vcf[0][1]).vcf.summary,
					path(process.out.vcf[0][1]).vcf.variantsMD5
```

File: modules/nf-core/lofreq/callparallel/tests/main.nf.test [hunk 3]
```diff
@@ -67,7 +67,7 @@ nextflow_process {
...
                     ).match() },
                 { assert path(process.out.vcf.get(0).get(1)).linesGzip.contains('##INFO=<ID=CONSVAR,Number=0,Type=Flag,Description="Indicates that the variant is a consensus variant (as opposed to a low frequency variant).">') }
```
Comment (famosab) [inline modules/nf-core/lofreq/callparallel/tests/main.nf.test:72]:
```suggestion
                    process.out.tbi,                    
                    path(process.out.vcf[0][1]).vcf.summary,
					path(process.out.vcf[0][1]).vcf.variantsMD5
                    ).match() },
```

**Review by famosab — CHANGES_REQUESTED** [review]
Looks good otherwise, this is just another improvement :)

File: modules/nf-core/lofreq/callparallel/tests/main.nf.test (hunk 2 above)
Comment (nevinwu) [reply modules/nf-core/lofreq/callparallel/tests/main.nf.test:37]: Oh I didn't really know this could be done this way.

**Comment by nevinwu** [comment]
@nf-core-bot fix linting pls.

**Comment by famosab** [comment]
Ok seems like the test files do not return any variants. That is why the linting fails -> we can change the `.vcf.variantsMD5` to `.vcf.summary`

File: modules/nf-core/lofreq/callparallel/tests/main.nf.test [hunk 4]
```diff
@@ -34,9 +34,9 @@ nextflow_process {
                 { assert process.success },
                 { assert snapshot(
                     process.out.versions,
-                    process.out.tbi.collect { file(it[1]).getName() }
+                    process.out.tbi,
+                    path(process.out.vcf[0][1]).vcf.variantsMD5
```
Comment (famosab) [inline modules/nf-core/lofreq/callparallel/tests/main.nf.test:38]:
```suggestion
                    path(process.out.vcf[0][1]).vcf.summary
```

File: modules/nf-core/lofreq/callparallel/tests/main.nf.test [hunk 5]
```diff
@@ -67,9 +67,9 @@ nextflow_process {
...
```
Comment (famosab) [inline modules/nf-core/lofreq/callparallel/tests/main.nf.test:71]:
```suggestion
                    path(process.out.vcf[0][1]).vcf.summary
```

**Review by famosab — APPROVED** [review]
We might need to adapt to other testdata that actually has variants in it, if we have that we can use `.variantsMD5` again. But for now I think this looks good enough :)

**Comment by nevinwu** [comment]
Yes, it makes sense. Thank for your review and your feedback @famosab! I still need to learn and adapt to `nf-test`.

**Pull request merged**